ingestion/tests/magen_ingestion_test.py

```diff
@@ -1939,13 +1948,15 @@ def test_post_file_share_Fail_KS_error(self):
                                              json_body=ks_get_resp_json_obj,
                                              response_object=None)
             get_mock = Mock(return_value=get_rest_return_obj)
+            mock_value = Mock(return_value=MongoReturn(count=1))
             with patch('magen_rest_apis.rest_client_apis.RestClientApis.http_get_and_check_success', new=get_mock):
-                jquery_file_share_url = server_urls_instance.ingestion_server_base_url + "file_share/"
+                with patch('magen_user_api.user_model.UserModel.select_by_email', new=mock_value):
+                    jquery_file_share_url = server_urls_instance.ingestion_server_base_url + "file_share/"
 
-                # pass wrong asset_id here
-                file_share_resp_obj = type(self).app.post(jquery_file_share_url, data={'file': '9c7b005-f027-4d6f-bea3-c61dec6e50',
-                                                                                      'selected_user': ['Bob']},
-                                                          headers={'content-type': 'multipart/form-data'})
+                    # pass wrong asset_id here
+                    file_share_resp_obj = type(self).app.post(jquery_file_share_url, data={
+                        'asset_id': '9c7b005-f027-4d6f-bea3-c61dec6e50', 'users': 'Bob'},
```

Comment:
let's use more explicit 'wrong' asset_id. Either store this one in a variable or pass a human-friendly string.